.github/workflows/registry_upload.yml

```diff
@@ -0,0 +1,84 @@
+#
```

Comment:
Would we like to run the tests in the pipeline from the published images instead of those built inside the pipeline?